### PR TITLE
Remove "blog" tutorial (#7786)

### PR DESCRIPTION
**Pull request by jtheuerkauf**
* Completely remove docs for the blog tutorial - caused odd page continuity
* Add tips for using bake to create structures and migrations
* Split database.rst so the Articles model is a separate page
* Minor grammar and phrasing updates

### CODE REVIEW

File: en/tutorials-and-examples/cms/database.rst [hunk 1]
```diff
@@ -153,80 +153,58 @@ able to connect to the database' section has a green chef hat.
...
+==========
 
-Models are the heart of CakePHP applications. They enable us to read and
-modify our data. They allow us to build relations between our data, validate
-data, and apply application rules. Models provide the foundation necessary to
-create our controller actions and templates.
+The SQL statements to create the tables for this tutorial can also be generated
+using the Migrations Plugin. Migrations provide a platform-independent way to
+run queries so the subtle differences between MySQL, PostgreSQL, SQLite, etc.
+don't become obstacles.
```
Comment (jtheuerkauf) [inline en/tutorials-and-examples/cms/database.rst:162]: Bear in mind, all these changes that look like complete replacements are actually new content, and the original was split to the new `articles-model.rst` file.

**Comment by markstory** [comment]
Thank You :tada:

**Comment by neuberkauf** [comment]
Glad to help, glad you liked the changes.

**Comment by jtheuerkauf** [comment]
Glad to help, glad you liked the changes.

**Pull request merged**